modular/translations/code/translation_data/ru_names.toml

```
@@ -14495,6 +14782,30 @@ instrumental = "изоляционными перчатками"
 prepositional = "изоляционных перчатках"
 gender = "plural"
 
+["Insulated Gloves (Yellow/Tan)"] # unique vendor name
+nominative = "жёлтые изоляционные перчатки"
+
+["Insulated Gloves (Black)"] # unique vendor name
+nominative = "чёрные изоляционные перчатки"
```

Уникальные имена с большой буквы, ведь они кейс-сенситив
```suggestion
nominative = "Чёрные изоляционные перчатки"
```

---

через код это решено, в toml все с маленькой чтобы потом не выискивать это по всем файлам